keps/sig-api-machinery/0006-apply.md

```diff
@@ -143,6 +143,29 @@ What are some important details that didn't come across above.
 Go in to as much detail as necessary here.
 This might be a good place to talk about core concepts and how they releate.
 
+#### Kubectl
+##### Server-side Apply
+Since server-side apply is currently in the Alpha phase, it is not
+enabled by default on kubectl. To use server-side apply on servers
+with the feature, run the command 
+`kubectl apply --experimental-server-side ...`.
+
+If the feature is not available or enabled on the server, the command
+will fail rather than fall-back on client-side apply due to significant
+semantical differences.
+
+As the feature graduates to the Beta phase, the flag will be renamed,
+changing the command to `kubectl apply --server-side ...`.
+
+Once the feature is stable (after the beta phase), it will not be the
```

Comment:
I'm confused by this sentence, are you sure that's what you meant?

A couple of things:
1. We're probably going to make apply default to server-side apply while in beta phase,
2. We're are indeed going to change the default value of the flag for apply,
3. We don't intend to change the flag after `--server-side`

Reply:
Thanks for the clarifications.

Based on those comments I basically removed one paragraph and reworded the last bit. Hopefully that helps